ZenPacks/zenoss/ZenJMX/pom.xml

```diff
@@ -88,6 +88,13 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.8.1</version>
+        <configuration>
+            <compilerArgs>
+                <args>-parameters</args>
+            </compilerArgs>
+            <source>11</source>
+            <target>11</target>
```

Comment:
Please test having these values set to 8.
A value of 8 means compatible with Java 8 which is the version of Java we've been shipping with as far back as ZSD 5.2.0.

If this does work, please update the `COMPAT_ZENOSS_VERS` parameter in `setup.py` to show 5.2.0 as the minimum supported version.

Reply:
Changed to 8, build doesn't fail